### ISSUE TITLE
Cannot read property 'getAttributeNames' of null

### ISSUE THREAD

**Issue by erikscheepers** [livewire/livewire#302]
**Describe the bug**
When polling a template I get this error in my console, and the polling stops:

> Uncaught (in promise) TypeError: Cannot read property 'getAttributeNames' of null
    at _default.extractTypeModifiersAndValue (livewire.js?id=1387c89dcf270acf8587:3411)
    at new _default (livewire.js?id=1387c89dcf270acf8587:3379)
    at new DOMElement (livewire.js?id=1387c89dcf270acf8587:3552)
    at DOMElement.closestByAttribute (livewire.js?id=1387c89dcf270acf8587:3712)
    at DOMElement.closestRoot (livewire.js?id=1387c89dcf270acf8587:3707)
    at onNodeAdded (livewire.js?id=1387c89dcf270acf8587:1968)
    at callHook (livewire.js?id=1387c89dcf270acf8587:4057)
    at handleNodeAdded (livewire.js?id=1387c89dcf270acf8587:4193)
    at handleNodeAdded (livewire.js?id=1387c89dcf270acf8587:4209)
    at morphEl (livewire.js?id=1387c89dcf270acf8587:4393)

**To Reproduce**
(cleaned up for simplicity, just to get the error)
```php
class Polling extends Component
{
    public function mount($model_id)
    {
//
    }

    public function hydrate()
    {
//
    }

    public function render()
    {
        return view('livewire.polling', []);
    }
}
```
My template file:
```html
<div wire:poll.800ms="$refresh">
    <div>
        <table class="class" id="div">
        </table>
    </div>
</div>
```
Upon changing my template file (during polling) to below I get the aforementioned error
```html
<div wire:poll.800ms="$refresh">
    <div></div>
    <div>
        <table class="class" id="div">
        </table>
    </div>
</div>
```

**Expected behavior**
I expect the template to just update to the new format and polling continues

**Comment by erikscheepers** [comment]
Giving one or both of the div's an id prevents the error described above:
```html
<div wire:poll.800ms="$refresh">
    <div id="this"></div>
    <div id="that">
        <table class="class" id="div">
        </table>
    </div>
</div>
```
Don't know if this is a fixable issue or if I'm stupid and should always give my div's an id?

**Comment by rydurham** [comment]
This is related to #285

**Comment by calebporzio** [comment]
This has to do with the way the dom-differ inside Livewire behaves (morphdom).

The error is being caused because a `<div>` is being added to the tree in the same place as a `<div>` in the old tree. Morphdom decides to update the div and append the new one after it, instead of being smart enough to recognize that, though they are both divs, the very next div in the new tree is an exact match to the div in the old tree.

I know that's a pretty dense explanation, but I'm putting it here for posterity's sake.

The fix for this should be added in v0.3.13
